### PR TITLE
Remove deprecated assets from release branch

### PR DESCRIPTION
**Pull request by elesel**
Test run: https://github.com/Azure/azureml-assets/actions/runs/4951081231

### CODE REVIEW

**Review by vizhur — APPROVED** [review]

**Review by novaturient95 — APPROVED** [review]

File: scripts/azureml-assets/azureml/assets/asset_utils.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (novaturient95) [inline scripts/azureml-assets/azureml/assets/asset_utils.py:1]: thanks @elesel for quick change

File: scripts/azureml-assets/azureml/assets/asset_utils.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    """List assets.
+
+    Args:
+        args (argparse.Namespace): Args from argparse.
+    """
+    # Find assets under input dir
+    asset_list = []
+    for asset_config in util.find_assets(args.input_dirs, args.asset_config_filename):
+        asset_list.append(asset_config.partial_name)
+    asset_list.sort()
```
Comment (lykelly19) [inline scripts/azureml-assets/azureml/assets/asset_utils.py:88]: Does this need to account for case-insensitivity?

Comment (elesel) [reply scripts/azureml-assets/azureml/assets/asset_utils.py:88]: Nah, it's just for convenience.

File: scripts/azureml-assets/azureml/assets/asset_utils.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+                             output_directory_root=output_directory_root,
+                             release_directory_root=release_directory_root,
+                             use_version_dir=use_version_dirs)
+        if version:
+            logger.print(f"Copied {asset_config.type.value} {asset_config.name} version {version}")
+            copied_count += 1
+    logger.print(f"{copied_count} of {asset_count} asset(s) copied")
+
+    # Set variables
+    logger.set_output(DELETED_COUNT, copied_count)
```
[the remark this answers is not in the document]
Comment (elesel) [reply scripts/azureml-assets/azureml/assets/asset_utils.py:75]: Actually, this code shouldn't even be here! I forgot to clean up following a copy and paste from another file.

[the remark this answers is not in the document]
Comment (elesel) [reply scripts/azureml-assets/azureml/assets/asset_utils.py:75]: See #642

**Pull request merged**